docker-compose.override.yml

```diff
@@ -20,8 +20,15 @@ services:
     depends_on:
       - watch-gulp
   ui:
+    build:
+      context: .
+      target: gulp
```

Comment:
It's possible to proxy BrowserSync through to Nginx, but easier to just replace it.